rules/mastg-android-webview-bridges.yml

```diff
@@ -0,0 +1,12 @@
+rules:
+  - id: mastg-android-webview-bridges
+    severity: WARNING
+    languages:
+      - java
+    metadata:
+      summary: This rule detects WebView javascript - native bridges.
+    message: "[MASVS-PLATFORM-2] Detected javascript bridge in WebView."
+    pattern-either:
+      - pattern: $WEBVIEW.setJavaScriptEnabled(true)
+      - pattern: $WEBVIEW.addJavascriptInterface($BRIDGE, $NAME)
+      - pattern: "@JavascriptInterface $RETURN_TYPE $FUNCTION (...){...}"
```

Comment:
I tried to match the whole Bridge class with a pattern like the following, but without much success
```yml
  patterns:
    - pattern: $_.addJavascriptInterface(new $BRIDGE (...), $_)
    - pattern: class $BRIDGE { ... }
```